photon-client/src/main/scala/com/linkedin/photon/ml/estimators/GameEstimatorEvaluationFunction.scala

```diff
@@ -46,11 +46,16 @@ class GameEstimatorEvaluationFunction(
   /**
    * Performs the evaluation.
    *
-   * @param hyperParameters The vector of hyperparameter values under which to evaluate the function
+   * @param candidate The candidate vector of hyperparameter with values in [0, 1]
    * @return A tuple of the evaluated value and the original output from the inner estimator
    */
-  override def apply(hyperParameters: DenseVector[Double]): (Double, GameResult) = {
-    val newConfiguration = vectorToConfiguration(hyperParameters)
+  override def apply(candidate: DenseVector[Double]): (Double, GameResult) = {
+
+    // Rescale candidate
+    candidate *= (range.end - range.start)
+    candidate += range.start
```

Comment:
To reiterate the comment from the Driver. I think you should scale it by rangeWrapped since you are rescaling the priors by range wrapped. And then take the exp. That is [0,1] -> [ -3, 3] -> [0.001, 1000] instead of [0,1] -> [0.001, 1000]

Reply:
Yes. Actually the variable `range` should always have [-3, 3], never should be [0.001, 1000]. This is true in XGBoost. However in GAME, we use a "LOG" by default and specify 0.001--1000 in configuration. That's the reason for having rangeWrapped, which takes the role of range.